wrapper/src/main/java/software/amazon/jdbc/plugin/cache/DataRemoteCachePlugin.java

```diff
@@ -140,33 +147,76 @@ private ResultSet cacheResultSet(String queryStr, ResultSet rs, int expiry) thro
 
   /**
    * Determine the TTL based on an input query
-   * @param queryHint string. e.g. "NO CACHE", or "cacheTTL=100s"
+   * @param queryHint string. e.g. "CACHE_PARAM(ttl=300s)" or "CACHE_PARAM(ttl=100s, key=custom)"
```

Comment:
Just need to mention the 2nd scenario. The first one is a subset of the second scenario where multiple parameters are specified.